src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -160,6 +161,12 @@ def __init__(self, *args, **kwargs):
 
         self.agent_sessions = None
         self.setup_start_servers = True
+        self.orterun_env = None
```

Comment:
Unless I missed it, it looks like `daos_core_base.py` is the only one using this when it uses orterun to run daos_test in `run_subtest()`.  If so, it should be removed from here.

Reply:
This is used by all the three *base.py files for client logs